### PR TITLE
bump version for 5.1.5

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/12393
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: statistics.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ TiDB uses statistics to decide [which index to choose](/choose-index.md). The `t
 | Version | Default value |
 | :- | :- |
 | v5.1.0 - v5.1.3 | `2`, which serves as an experimental feature |
-| v5.1.4 and later v5.1.x versions | `1` |
+| v5.1.5 and later v5.1.x versions | `1` |
```
Comment (hfxsd) [inline statistics.md:13]:
```suggestion
| v5.1.4 and later v5.1.x versions | `1` |
```

File: system-variables.md [hunk 2]
```diff
@@ -283,7 +283,7 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 | Version | Default value |
 | :- | :- |
 | v5.1.0 - v5.1.3 | `2`, which serves as an experimental feature |
-| v5.1.4 and later v5.1.x versions | `1` |
+| v5.1.5 and later v5.1.x versions | `1` |
```
Comment (hfxsd) [inline system-variables.md:286]:
```suggestion
| v5.1.4 and later v5.1.x versions | `1` |
```

**Comment by shichun-0415** [comment]
The version in the following files also needs update:
image
image

**Comment by hfxsd** [comment]
> The version in the following files also needs update: image image

ok  updated

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: releases/release-notes.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ title: Release Notes
 
 ## 5.1
 
+- [5.1.5](/releases/release-5.1.5.md)
```
Comment (hfxsd) [inline releases/release-notes.md:9]:
```suggestion
```

**Pull request merged**